### PR TITLE
 + Sound Config

### PR DESCRIPTION
**Pull request by winer222**
+ Passenger Cabin

(fixes for #64)

### CODE REVIEW

File: OmsiHook/OmsiSoundPack.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class OmsiSoundPack : OmsiObject
         }*/
         public string FileName
         {
-            get => Memory.ReadMemoryString(Memory.ReadMemory<int>(Address + 0x8));
+            get => Memory.ReadMemoryString(Address + 0x8);
             set => Memory.WriteMemory(Address + 0x8, value);
```
Comment (space928) [inline OmsiHook/OmsiSoundPack.cs:19]: If you've changed the getter have you checked that the setter still works?

File: OmsiHook/OmsiSoundPack.cs [hunk 2]
```diff
@@ -25,7 +25,7 @@ public bool AI
         }
         public string Path
         {
-            get => Memory.ReadMemoryString(Memory.ReadMemory<int>(Address + 0x10));
+            get => Memory.ReadMemoryString(Address + 0x10);
             set => Memory.WriteMemory(Address + 0x10, value);
```
Comment (space928) [inline OmsiHook/OmsiSoundPack.cs:29]: Ditto

File: OmsiHook/OmsiSoundPack.cs (hunk 1 above)
Comment (winer222) [reply OmsiHook/OmsiSoundPack.cs:19]: seems to work

File: OmsiHook/OmsiSoundPack.cs (hunk 2 above)
Comment (winer222) [reply OmsiHook/OmsiSoundPack.cs:29]: ditto

**Pull request merged**